kuma/core/managers.py

```diff
@@ -241,6 +243,16 @@ def resolve_allowed_tags(model_obj, tags_curr, tags_new,
     return tags_out
 
 
+class IPBanManager(models.Manager):
+    def active(self, ip):
+        return super(IPBanManager, self).filter(ip=ip, deleted__isnull=True)
+
+    def delete_old(self, days=30):
+        cutoff_date = date.today() - timedelta(days=days)
+        old_ip_bans = self.get_query_set().filter(created__lte=cutoff_date)
```

Comment:
`self.filter(..)` is enough, the call to `get_query_set` is implied when using `self`